lua/example_of_dynamic_config_request.lua

```diff
@@ -5,4 +5,51 @@
 -- Functions to process the GT policies.
 -- ......
 
-return dy_default_proc()
+local dylib = require("dylib")
+local ffi = require("ffi")
+
+dyc = ffi.cast("struct dynamic_config *", dy_conf_ptr)
```

Comment:
This cast should be done in `lua/dylib.lua` since it's very useful and all messages will likely need it.